calculator/operations.py

```diff
@@ -1,18 +1,32 @@
 
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    # need to handle len 0 and len 1
+    x = args[0]
+    for i in args[1:]:
+        x -= i
+    return x
 
 def multiply(*args):
-    # your implementation here
-    pass
+    # need to handle len 0 and len 1
+    # what about floating point?
+    x = args[0]
+    for i in args[1:]:
+        x *= i
+    return x
 
```

Comment:
return reduce(lambda x, y: x*y, args)

and likewise for below for the divide:

return reduce(lambda x, y: x / float(y), args)

i think reduce is perfect to use here, only problem is in python3 you need to import it from functools